docs/architecture/mobile-clients/overview.md

```diff
@@ -16,5 +16,5 @@ differs:
 
 ## Presentation
 
-The presentation layer is implemented using `Xamarin.Forms` for the mobile apps, except for the
-watchOS one which uses `SwiftUI` [see ADR](../adr/watchOS-use-swift)
+The presentation layer is implemented using `.NET MAUI` for the mobile apps, except for the watchOS
```

Comment:
⛏️ "MAUI" or ".NET MAUI" aren't code references in my opinion and we shouldn't put backticks all over the place.

Reply:
@withinfocus Roger that, this also applies to `SwiftUI`, `XCode`, `CoreData`, `Xamarin.Forms`. At the moment I use that in order to make the string stand out. Would you like Bold/Italics more? Or simply put it in the raw way?

Reply:
I think undecorated is still best for uniform presentation.